resource/airflow_dag_pipeline.yaml

```diff
@@ -192,6 +192,7 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ((secrets.role-putter-secret-access-key))
         AWS_NODES_ROLE_ARN: ((secrets.nodes-role-arn))
         APP_NAME: ((app-name))
+        DELETE_AFTER_TEST: true
```

Comment:
How/Where is this used?

Reply:
In `put-iam-policy` this is used to delete the policy if it was created when testing with a `Deny` all statement appended.